server/routes/contacts/manage/contact-confirmation/contactConfirmationController.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express'
+import { PageHandler } from '../../../../interfaces/pageHandler'
+import { Page } from '../../../../services/auditService'
+import ContactsService from '../../../../services/contactsService'
+import { IsContactConfirmedSchema } from './contactConfirmationSchema'
+import { navigationForAddContactJourney } from '../../add/addContactFlowControl'
+import PrisonerJourneyParams = journeys.PrisonerJourneyParams
+
+export default class ContactConfirmationController implements PageHandler {
+  constructor(private readonly contactService: ContactsService) {}
+
+  public PAGE_NAME = Page.CONTACT_CONFIRMATION_PAGE
+
+  GET = async (
+    req: Request<PrisonerJourneyParams & { mode?: 'EXISTING' | 'NEW' }, unknown, unknown, { contactId?: string }>,
+    res: Response,
+  ): Promise<void> => {
+    const { journeyId, mode } = req.params
+    const { contactId } = req.query
+    const { prisonerDetails } = res.locals
+    const journey = req.session.addContactJourneys[journeyId]
+    journey.mode = mode
```

Comment:
I don't think this should be set here as it's already set in the mode controller. I guess the mode controller will redirect here instead of relationship now in a future PR?

Reply:
I thinks so because when selecting a contact from the search results the user will have to confirm if it is the right person or not first. If 'Yes' is selected then the next screen will be relationship otherwise the user is redirected back to the contact search results.